### ISSUE TITLE
Facing error : "nvidia-container-cli: requirement error: unsatisfied condition: brand = quadrortx: unknown"

### ISSUE THREAD

**Issue by lksharma10** [NVIDIA/nvidia-container-toolkit#24]
AWS - 'g4dn' Tesla T4 machine
Nvidia-driver base image - [nvidia/cuda:11.6.0-base-ubuntu18.04]

**We are observing a failure.**
```
[/usr/bin/nvidia-container-cli --load-kmods configure --ldconfig=@/sbin/ldconfig --device=GPU-76ac82dd-f6b6-90f5-155f-707b48143dc0 --compute --utility --require=cuda>=11.6
brand=tesla,driver>=418,driver<419
brand=tesla,driver>=450,driver<451
brand=tesla,driver>=470,driver<471
brand=unknown,driver>=470,driver<471
brand=nvidia,driver>=470,driver<471
brand=nvidiartx,driver>=470,driver<471
brand=quadrortx,driver>=470,driver<471
--pid=29236 /var/lib/docker/overlay2/e5e1ee1901573449b4467424d41f618ee9f94f819ebb3d67ab65cf542a6ebea9/merged]
nvidia-container-cli: requirement error: unsatisfied condition: brand = quadrortx: unknown
```

We found an option to disable require using `NVIDIA_DISABLE_REQUIRE`. But we would like to understand the root cause so that we can fix the issue without using this flag. Requesting your inputs/suggestions.

**Comment by klueska** [comment]
You need to be running a toolkit >= 1.9.0 for the quadrortx brand to be recognized, otherwise you will see the error without NVIDIA_DISABLE_REQUIRE.

**Comment by lksharma10** [comment]
We tried to update latest version of the toolkit, using this
`RUN distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
      && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
      && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
            sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
            tee /etc/apt/sources.list.d/nvidia-container-toolkit.list

RUN apt-get update

RUN apt-get install -y nvidia-docker2`

We are executing this in Kubernetes pods, and we are still encountering the issue.

**Comment by klueska** [comment]
Can you verify the version of `libnvidia-container` you have installed after this update?
```
$ dpkg -l | grep libnvidia-container1
ii  libnvidia-container1:amd64           1.10.0~rc.2-1                     amd64        NVIDIA container runtime library
```
It needs to be > 1.9.0

**Comment by lksharma10** [comment]
This issue is resolved by updating the Kubernetes EKS to 1.22. Thanks for all the help @klueska
